clappr/clappr-ios
# Fullscreen handle by application

### Summary
When user requests fullscreen, the player change the layout to fullscreen by itself.

### Goal
 - Create an option that will be used when the application needs to handle with fullscreen instead of player.
 - Fire an event to notify app that user request's fullscreen or exitFullscreen
 - Add option to test the option on ``Sample``

### How to test
 #### Test: Application controls fullscreen
 1. Run App and switch the option **fullscreen controled by app** to **on**.
 2. Open the video
 3. Change the orientation to landscape
 4. **The player should change the icon to fullscreen**
 5. Change the orientation to portrait
 6. **The player should change the icon to default(embed)**

#### Test: Player controls fullscreen
 1. Run App and switch the option **fullscreen controled by app** to **off**.
 2. Open the video
 3. Tap to set the video to fullscreen
 4. **The player should change the layout and icon to fullscreen**
 5. Tap to close fullscreen of video
 6. **The player should change the layout and icon to embed**
